gig/migrations/0013_gig_email_changes_historicalgig_email_changes_and_more.py

```diff
@@ -17,7 +17,6 @@ class Migration(migrations.Migration):
             old_name='send_update',
             new_name='email_changes',
         ),
-```suggestion
```

Comment:
I'm confused - is line 17 supposed to be there? My migrations are failing now.

Reply:
Line 20 was an error, I accidentally put that markdown in the change request. 

Your migration is failing because you ran an older version of the migration which added the column `email_changes`. I changed your migration file which now attempts to rename `send_update` to `email_changes`. This collides because you already ran the migration locally.  This problem is unique to your local environment.

Two ways forward from here:
* Revert the migration and re-run it after my change
* Drop your DB and re-run migrations from scratch